app/helpers/govuk_form_helper.rb

```diff
@@ -0,0 +1,131 @@
+module GovukFormHelper
+  # = GOVUK Form Tag Helpers
+  #
+  # For usage, components and patterns see:
+  # https://design-system.service.gov.uk/
+
+  # Can be used to wrap multiple or single components.
+  #
+  # Creates a div with the 'govuk-form-group' css class, if the provided object has errors, adds
+  # the 'govuk-form-group--error' css class.
+  #
+  # Passing the attribute symbol will only add the error class if the object contains errors for
+  # the matching symbol.
+  #
+  # Generally pass only an object when wrapping multiple components and pass an object and attribute when
+  # wrapping a single component.
+  #
+  # === Examples
+  # ==== Wrap multiple components
+  #
+  #   <%= form_group_tag object do %>
+  #     <fieldset class="govuk-fieldset">
+  #       <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+  #         <h1 class="govuk-fieldset__heading">
+  #           Where do you live?
+  #         </h1>
+  #       </legend>
+  #     ...
+  #     </fieldset>
+  #   <% end %>
+  #
+  #   <div class="govuk-form-group">
+  #     <fieldset class="govuk-fieldset">
+  #       <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+  #         <h1 class="govuk-fieldset__heading">
+  #           Where do you live?
+  #         </h1>
+  #       </legend>
+  #        ...
+  #     </fieldset>
+  #   </div>
+  #
+  # ==== Wrap a single component
+  #
+  #   <%= form_group_tag object, :event_name do %>
+  #     ...
+  #   <% end %>
+  #
+  #   <div class="govuk-form-group">
+  #     <label class="govuk-label" for="object_event_name">
+  #       Event name
+  #     </label>
+  #     <input class="govuk-input" id="object_event_name" name="event_name" type="text">
+  #   </div>
+  #
+  def form_group_tag(object, attribute = nil, &block)
+    css_classes = ["govuk-form-group"]
+    case attribute
+    when Symbol
+      css_classes += ["govuk-form-group--error"] if object.errors.key?(attribute)
```

Comment:
I think `<<` is preferred over `+=`, becuase it's faster. `+=` creates a new object, while `<<` appends onto the existing object

Reply:
Nice, thanks! And looks nicer!